CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+=======
```

Comment:
@ricknjacky remove these lines.

Reply:
can u tell me why this is happening because(chanelog.md) I didn't even touch this file its happening automatically for some reason, yes, I'll remove them